crates/jstz_cli/src/logs/mod.rs

```diff
@@ -9,36 +10,17 @@ mod trace;
 pub enum Command {
     /// View logs
     Trace {
-        /// View console.log messages
-        #[arg(long)]
-        log: bool,
-        /// View console.info messages
-        #[arg(long)]
-        info: bool,
-        /// View console.warn messages
-        #[arg(long)]
-        warn: bool,
-        /// View console.error messages
-        #[arg(long)]
-        error: bool,
-        /// View contract creations
-        #[arg(long)]
-        contract: bool,
-        /// Add custom search strings
-        #[arg(long, value_name = "custom_strings")]
-        custom: Vec<String>,
+        #[arg(value_name = "SMART_FUNCTIOIN")]
```

Comment:
SMART_FUNCTION

I think it would be best to use ADDRESS or SMART_FUNCTION_ADDRESS for consistency with other naming - e.g. self address